ansible/configs/ansible-automation-platform/roles/deploy_automationcontroller_cfg/defaults/main.yml

```diff
@@ -0,0 +1,26 @@
+---
+
+deploy_automationcontroller_installer_url: "" # e.g. http://example.com/ansible-automation-setup-bundle-latest.tar.gz"
```

Comment:
that default `""` won't work, right?
If it's the case, then i think it's preferred not to set it and make sure it's provided by the user using the role.